lib/crowbar/init/application.rb

```diff
@@ -31,26 +31,30 @@ class Application < Sinatra::Base
         set :environment, ENV["RAILS_ENV"]
       end
 
+      if settings.environment == :development
+        set :logpath, "#{settings.root}/log"
+      else
+        set :logpath, "/var/log/crowbar"
+      end
+
+      configure do
+        ::Logger.class_eval { alias :write :'<<' }
```

Comment:
It's a method ;)

See https://ruby-doc.org/stdlib-2.1.0/libdoc/logger/rdoc/Logger.html#method-i-3C-3C